.gitignore

```diff
@@ -23,3 +23,6 @@ vendor
 *.swp
 *.swo
 *~
+
+#Mac
+.DS_Store
```

Comment:
https://github.com/github/gitignore/blob/main/Global/macOS.gitignore  I think we can add more to gitignore

Reply:
Resolving this convo to unblock the change.